### DISCUSSION TITLE
What files should I keep in my version control

### DISCUSSION THREAD

**Discussion by AugustinSorel** [GoogleChromeLabs/bubblewrap#834] [Q&A] [answered]
Hello,

I have one of those silly question... 
So I am currently following [this guide](https://developers.google.com/codelabs/pwa-in-play#1) to build a pwa. 
I followed this guide and ran the commands at the root of my project, everything worked as expected.

What files should I keep in my version control ? This guide only mentions that I should keep the `twa-manifest.json`. But what about the `app-release-bundle.aab` and `app-release-signed.apk` ?

**Comment by andreban** [comment, marked as answer]
Hey @AugustinSorel ,

`app-release-bundle.aab` and `app-release-signed.apk` are the output build files and there's no need to keep them in your version-control, as you can rebuild them from `twa-manifest.json` and the upload keys. It's important to store the upload keys somewhere safe, but outside version control, as you don't want anyone to have access to them.

**Reply by AugustinSorel** [reply]
awesome thanks :smiley:
